src/store/conversationsStore.js

```diff
@@ -634,9 +634,8 @@ const actions = {
 			return
 		}
 
-		await setConversationUnread(token)
-		commit('updateUnreadMessages', { token, unreadMessages: 1 })
-		await dispatch('fetchConversation', { token })
+		const response = await setConversationUnread(token)
+		await dispatch('addConversation', response.data.ocs.data)
```

Comment:
Use `updateConversation` , although it is basically doing the same thing as `addConversation` for now but we can improve it later to patch the changes.

Reply:
I have no idea what I'm doing :D